README.md

```diff
@@ -36,13 +36,13 @@ By making a contribution to this project, you are deemed to have accepted the [D
 
 All conversations and communities on Medplum are expected to follow GitHub's [Community Guidelines](https://help.github.com/en/github/site-policy/github-community-guidelines)
 and [Acceptable Use Policies](https://help.github.com/en/github/site-policy/github-acceptable-use-policies). We expect
-discussions in issues and pull requests to stay positive, productive, and respectful. Remember: there are real people on
+discussions on issues and pull requests to stay positive, productive, and respectful. Remember: there are real people on
 the other side of the screen!
 
 #### Reporting a bug or proposing a new feature
 
-If you found a technical bug on Medplum or have ideas for features we should implement, the issue tracker is the best
-place to share with us. ([click here to open a new issue](https://github.com/medplum/medplum/issues/new))
+If you have found a technical bug on Medplum or have ideas for features we should implement, the issue tracker is the best
+place to share them with us. ([click here to open a new issue](https://github.com/medplum/medplum/issues/new))
```

Comment:
This change is unnecessary, and doesn't improve the clarity of the text; please revert it